src/main/scala/no/ndla/listingapi/model/domain/Card.scala

```diff
@@ -0,0 +1,34 @@
+package no.ndla.listingapi.model.domain
+
+import no.ndla.listingapi.ListingApiProperties
+import org.json4s.FieldSerializer
+import org.json4s.FieldSerializer._
+import org.json4s.native.Serialization._
+import scalikejdbc._
+
+case class ImageCoordinate(col: Long, row: Long)
+case class CoverPhoto(imageApiId: Long, cropStart: ImageCoordinate, cropEnd: ImageCoordinate)
+case class Label(`type`: Option[String], labels: Seq[String])
+case class Card(id: Option[Long],
```

Comment:
Godt poeng. Jeg heller har ingen bedre forslag. Føler "thing" eller "metadata" blir litt for abstrakt, men `docket` som @tanjag foreslår er kanskje bedre enn `Card`.